frontend/src/admin/components/sidebar/dropdown/dropdown-menu.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from "react";
+import MenuIcon from "./menu-icon";
+import DropdownMenuItem from "./dropdown-menu-item";
+import "./dropdown-menu.scss";
+
+export default function DropdownMenu() {
+    const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+    const toggleDropdown = () => {
+        setShowDropdown(!showDropdown);
+    };
+
+    const dismissHandler = (event: React.FocusEvent<HTMLButtonElement>): void => {
+        if (event.currentTarget === event.target) {
+            setShowDropdown(false);
+        }
+    };
+
+    return (
+        <div className="c-dropdown-menu">
+            <button
+                className={showDropdown ?"c-dropdown-menu__button--active" : "c-dropdown-menu__button"}
+                onClick={() => toggleDropdown()}
+                onBlur={(e: React.FocusEvent<HTMLButtonElement>): void => dismissHandler(e)}
+            >
+                <MenuIcon/>
+                <div className="c-dropdown-menu-items">
+                    <DropdownMenuItem id="download" name="Download user"/>
+                    <DropdownMenuItem id="upload" name="Upload user"/>
+                    <DropdownMenuItem id="add" name="Add user"/>
```

Comment:
You are specifying each item manually. That is not very dynamic. I read Anders comment that I think made you do this, but it seems like you might be misunderstanding what he meant. He still wants you to have a list that you can iterate over when making the items. He just didn't want a generic component like `DropdownMenu` to be hardcoded to the `Actions` list. Instead he wanted you to receive the list as a prop so you can keep the generic sounding component `DropdownMenu` generic. 

But I think we need to have a talk about naming component generic names. Refer to my other comment about this topic.

Reply:
✅ This is changed  - i have implemented the children props